### DISCUSSION TITLE
Gas mixing with immobile water in PorousFlow module

### DISCUSSION THREAD

**Discussion by sinaomrani96** [idaholab/moose#23920] [Q&A Modules: General]
Dear all,
Hope you are doing well.

I wanted to simulate a 2-phase system of immobile water and two gases. At first gases are separated and then they get mixed. I am using PorousFlow module, and the problem is that I cannot define fluid properties for each component, and it only accepts for phases. How I can define material property for each gas in my system since they are after all in the same phase. I was wondering if can be done with PorousFlow.

Thanks,
Sina

**Comment by sinaomrani96** [comment]
Dear all,
Hope you are doing well.

I was wondering if there is a solution for this or if not how the code should be modified. Thanks again for your insight.

Sina

**Reply by GiudGiud** [reply]
@cpgr

**Reply by GiudGiud** [reply]
@WilkAndy @cpgr
